src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/SuccessUpdater.kt

```diff
@@ -1,12 +1,33 @@
 package uk.gov.justice.digital.hmpps.hmppstier.service
 
+import com.amazonaws.services.sns.AmazonSNSAsync
+import com.google.gson.Gson
+import org.springframework.beans.factory.annotation.Value
 import org.springframework.stereotype.Service
-import uk.gov.justice.digital.hmpps.hmppstier.client.CommunityApiClient
-import uk.gov.justice.digital.hmpps.hmppstier.dto.TierDto
+import java.util.UUID
 
 @Service
-class SuccessUpdater(val client: CommunityApiClient) {
-  fun update(tierDto: TierDto, crn: String) {
-    client.updateTier("${tierDto.protectLevel.value}${tierDto.changeLevel.value}", crn)
+class SuccessUpdater(
+  private val amazonSNS: AmazonSNSAsync,
+  @Value("\${hmpps-events.topic}")private val topic: String,
+  private val gson: Gson
+) {
+
+  fun update(crn: String, calculationId: UUID) {
+    val tierChangeEvent = TierChangeEvent(EventType.HMPPS_TIER_CALCULATION_COMPLETE, crn, calculationId)
+    val message = Message(gson.toJson(tierChangeEvent), calculationId.toString())
```

Comment:
It looks like we are toJsoning tierChangeEvent twice - once on this line and then on the next. Do we need to?

Reply:
I _think_ so because the message is a json object and the parcel is also a json object with a string message.